source/src/PascalCompiler/Grammar/Nonterminals.h

```diff
@@ -9,204 +9,190 @@
+    Index,                           // 135
+    InputOperator,                   // 136
+    Label,                           // 137
+    LabelOrAssignmentStatement,      // 138
+    LabelSection,                    // 139
+    LeftHandVar,                     // 140
+    LocalDeclaration,                // 141
+    LogicAnd,                        // 142
+    LogicOr,                         // 143
+    LoopDirection,                   // 144
+    LoopWithParameterOperator,       // 145
+    Massive,                         // 146
+    MulExpression,                   // 147
+    NextAdditiveExpression,          // 148
+    NextConstDefinition,             // 149
+    NextGcdExpression,               // 150
+    NextIndex,                       // 151
+    NextLabel,                       // 152
+    NextLogicAnd,                    // 153
+    NextLogicOr,                     // 154
+    NextMulExpression,               // 155
+    NextRecordFieldDefinition,       // 156
+    NextRelationalExpression,        // 157
+    NextTypeDefinition,              // 158
+    NextVarDefinition,               // 159
+    NotEmptyExprList,                // 160
+    NotEmptyGlobalDeclaration,       // 161
+    NotEmptyIdList,                  // 162
+    NotEmptyLocalDeclaration,        // 163
+    NotEmptyResultParameters,        // 164
+    NotEmptyStatement,               // 165
+    NumericConstant,                 // 166
+    OneTypeParameters,               // 167
+    OperatorOverloadSection,         // 168
+    OutputOperator,                  // 169
+    OverloadBinaryOperator,          // 170
+    OverloadOperator,                // 171
+    OverloadOperatorEnd,             // 172
+    ProcedureDeclaration,            // 173
+    ProcedureParameters,             // 174
+    Program,                         // 175
+    RationalComplexPart,             // 176
+    RationalNumberPart,              // 177
+    RecordFieldDefinition,           // 178
+    RecordType,                      // 179
+    RelationalExpression,            // 180
+    ResultParameters,                // 181
+    RightHandVar,                    // 182
+    Statement,                       // 183
+    StatementList,                   // 184
+    Type,                            // 185
+    TypeDefinition,                  // 186
+    TypeSection,                     // 187
+    UnaryAdditiveOperator,           // 188
+    UnaryNotAdditiveOperator,        // 189
+    UnaryOperator,                   // 190
+    UnaryOperatorOverload,           // 191
+    UnconditionalJumpOperator,       // 192
+    UnsignedFactor,                  // 193
+    VarDefinition,                   // 194
+    VarHandlerList,                  // 195
+    VarSection,                      // 196
+    NONTERMINAL_FINISH_INDEX,        // 197
+    ACTION_START_INDEX,              // 198
+    ACTION1,                         // 199
+    ACTION3,                         // 200
```

Comment:
We don't have any uses of `ACTION1` and `ACTION3` so I suggest that we can safely remove them from enum.